### DISCUSSION TITLE
paddlespeech window 下运行失败 “ OSError: Invalid enum backend type `64`. ”

### DISCUSSION THREAD

**Discussion by dogvane** [PaddlePaddle/PaddleSpeech#2962] [General]
Log日志：

(pspeech) PS C:\ai\PaddleSpeech> paddlespeech tts --input "你好，欢迎使用百度飞桨深度学习框架！" --output ./output.wav
C:\Users\dogvane\.conda\envs\pspeech\lib\site-packages\librosa\core\constantq.py:1059: DeprecationWarning: `np.complex` is a deprecated alias for the builtin `complex`. To silence this warning, use `complex` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.complex128` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  dtype=np.complex,
C:\Users\dogvane\.conda\envs\pspeech\lib\site-packages\_distutils_hack\__init__.py:33: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
OSError: Invalid enum backend type `64`.
  [..\paddle/phi/common/backend.h:140]

windows版本：win10专业工作站版  2022H2
python: conda 安装 python 3.8.13

安装时间：20230227
cuda: 11.3

安装方式：
仅使用
pip install paddlepaddle -i https://mirror.baidu.com/pypi/simple
pip install pytest-runner  -i https://mirror.baidu.com/pypi/simple
pip install paddlespeech  -i https://mirror.baidu.com/pypi/simple

安装过程中未报任何错误，仅执行示例命令出现  “ OSError: Invalid enum backend type `64`. ”
paddlespeech   更换为git clone 代码，安装正常，但结果仍然是一样的错误。

**Comment by dogvane** [comment]
已安装 vs2022
内包含 v143 c++ 的c++编译模块

**Comment by yt605155624** [comment]
check https://github.com/PaddlePaddle/Paddle/issues/50300

**Comment by dogvane** [comment]
ok， 对 paddlepaddle 降级到 2.3.2 解决问题。当前默认安装的最新版 2.4.3 不匹配。
使用命令：
python -m pip install paddlepaddle==2.3.2 -i https://pypi.tuna.tsinghua.edu.cn/simple

补充一下
cpu x79 
2650v2

希望后面的2.4版本能直接运行。

**Comment by dogvane** [comment]
gpu版本， 2.3.2 没问题
 python -m pip install paddlepaddle-gpu==2.3.2.post112 -f https://www.paddlepaddle.org.cn/whl/windows/mkl/avx/stable.html

gpu版本，2.4.1 OSError: Invalid enum backend type `63`.
 python -m pip install paddlepaddle-gpu==2.4.1.post112 -f https://www.paddlepaddle.org.cn/whl/windows/mkl/avx/stable.html

看了一下其他issue，这个问题存在时间不短了。
